en-eng/Plugin-Configuration.md

```diff
@@ -2,16 +2,17 @@
 
 In Domoticz, go in Setup&gt;Hardware, in Type select &quot;Zigate plugin&quot;.
 
-[[File:https://github.com/pipiche38/Domoticz-Zigate-Wiki/blob/master/Images/Zigate-Configuration.png|frame|none|alt=|caption ]]
+![Plugin Main Screen](https://github.com/pipiche38/Domoticz-Zigate-Wiki/blob/master/Images/Plugin_main_screen.png)
+
 
 * Name: Select a name for your hardware (here it's zigate)
 * Type: Zigate plugin
 * Data Timeout: Disabled
-* Select your model (USB or Wifi)
+* Select your model (USB or Wifi or PI)
 ** if Wifi enter IP of your Zigate, keep port to 9999 (not possible to change this on the zigate yet)
 ** if USB, select your device port (always /dev/ttyUSBx under linux, COMx for Windows)
 * Software Reset: False ( If you want to do a Software reset of the Zigate at startup, you can set to True
-
+* Rescan Group Membership: By default the plugin do not scan for group membership, it relys on the what was done before. So If you have remove/add new devices , you can set this flag to True to force the Membership scan. In such case, **make sure that all devices are powered on** otherwise you might face some difficulties.
```

Comment:
* Rescan Group Membership: By default the plugin does not scan for group membership, it relies on the what was done before. So If you have removed/added new devices , you can set this flag to "True" to force the Membership scan. In such case, **make sure that all devices are powered on** otherwise you might face some difficulties.